tests/settings.py

```diff
@@ -16,6 +16,7 @@
     'django.contrib.staticfiles',
     'openwisp_utils.admin_theme',
     'django.contrib.sites',
+    'openwisp_utils',
```

Comment:
is this really necessary? It should not be right?

Isn't drf-yasg needed? If it is, don't forget to mention it in the README section you're adding.

Reply:
I'm confused, do you know why adding the line to urls.py fails?

https://travis-ci.org/github/openwisp/openwisp-utils/jobs/685020239#L405

Reply:
In my experience, that may be indicating another import error is being detected in that file.

I would create a new virtualenv and execute all the commands in `.travis.yml` (exactly in the order used by travis) to replicate its behavior and find out if the error is replicated.

Reply:
Looks like the problems was that we need to run `pip install -e .`... as the last command otherwise openwisp-utils from pip is installed which does not contains the urls.py file